appveyor.yml

```diff
@@ -11,7 +11,7 @@ environment:
     CSL_SYSTEM_TAG: "win64"
     CACHE_DIR: "c:\\c"
     WORK_DIR: "c:\\w"
-    DCONFIG: "qanet_tns_wallet"
+    DCONFIG: "ci_wallet"
```

Comment:
Will this prefix be stripped further? There is no `testnet_ci_wallet` in `constants.yaml`.